cmsranking/Config.py

```diff
@@ -132,7 +138,9 @@ def _load_many(self, conf_paths: list[str]) -> bool:
                                 conf_path, errno.errorcode[error.errno],
                                 os.strerror(error.errno))
                 return False
-        logger.warning("No config file found, using hardcoded defaults.")
+        tried = " ".join(conf_paths)
+        logging.warning(f"No configuration file found, tried: {tried}. "
+                        "Falling back to default values.")
```

Comment:
At some point we discussed this being a hard failure instead of a warning.

IMO it's a good idea as the default values are almost never what you want (either for CMS or for RWS), but it might be worth doing separately.

Reply:
Agreed.